### ISSUE TITLE
normalization

### ISSUE THREAD

**Issue by simpleishappy** [cookcodes/percepnet#2]
Thanks for sharing your code with us!
May I ask how to get the parameters in the normalization.h, or what does it mean?

**Comment by cookcodes** [comment]
Before feed the data to neural network, the data are nomalized. It will help to reduce the loss of training result. The mean and variance data are statistic result from all training data.

**Comment by simpleishappy** [comment]
Thank you for the explanation, but the normalization should refer to standardization？

**Comment by cookcodes** [comment]
You are right, it should be standardization instead of normalization.
